src/interactive/runtime.tsx

```diff
@@ -1,21 +1,42 @@
 import * as React from "react";
 import { IRuntimeInitInteractive, useInteractiveState, setHint } from "@concord-consortium/lara-interactive-api";
 
-import { defaultAuthoredState, IAuthoredState, IInteractiveState } from "./types";
+import { defaultAuthoredState, IAuthoredState, IInteractiveState, SensorRecording } from "./types";
 import { App } from "../components/app";
 
 interface Props {
   initMessage: IRuntimeInitInteractive<IInteractiveState, IAuthoredState>;
 }
 
+
+const getRecordings = (useAuthoredData: boolean, recordedData?: SensorRecording): SensorRecording[] => {
+  if (!recordedData) {
+    return [];
+  }
+  const recording = {...recordedData};
+  // remove the data from the recording if we are just specifying the the sensor
+  // names, units, min/max values, etc. The authored data might include data
+  // from a previous save, we retain that, but we don't want to show it.
+  if (!useAuthoredData) {
+    recording.data = [];
+  } else {
+    recording.max = recording.data.reduce((max, point) => Math.max(max, point[1]), recording.max);
```

Comment:
The starting value of this reducer is the previous max value - is that intended?